integration/setup/setup.go

```diff
@@ -115,6 +115,11 @@ func installResources(ctx context.Context, config Config) error {
 					"Registry": map[string]interface{}{
 						"Domain": "quay.io",
 					},
+					"GiantSwarm": map[string]interface{}{
+						"AppOperator": map[string]interface{}{
+							"PauseAnnotation": "app-operator.giantswarm.io/paused",
```

Comment:
The name looks so generic now that I'm thinking we can put it as a default in `values.yaml` and skip `installations`. I think this pause annotation is a sane feature to have and having it defined in `values.yaml` would make it easily discoverable for external the user.